cvat/apps/engine/models.py

```diff
@@ -19,6 +19,17 @@ def get_prep_value(self, value):
             return value[:self.max_length]
         return value
 
+class DimensionType(str, Enum):
+    THREED = '3d'
```

Comment:
Something like `DIM_2D` and `DIM_3D` is better.

Reply:
Changed code accordingly.